goinstant/README.md

```diff
@@ -48,3 +48,13 @@ bash ./buildreleases.sh
 To build releases, create an instant tag and a release, the GitHub actions will build the code after creation of the release and add the binaries to the assets of the release if the GO CLI build succeeds.
 
 Check [here](https://github.com/openhie/instant/actions/new) to review the output of the build and status of the binary deploy.
+
+### Testing
+
+The [godog library](https://github.com/cucumber/godog), which provides us with the [Cucumber Framework](https://cucumber.io/) is used for functional tests of the Go CLI. To run the tests install godog and then navigate to the `goinstant` root folder and run the command below.
```

Comment:
The one sentence here is a little difficult to read... try replacing with the following:

"To run the tests, install godog, then navigate to the `goinstant` root folder. Finally, run the command below."